app/src/main/java/com/swent/assos/ui/screens/assoDetails/AssoDetails.kt

```diff
@@ -274,6 +275,22 @@ fun AssoDetails(assoId: String, navigationActions: NavigationActions) {
               modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp))
         }
       }
+
+      item {
+        Text(
+            text = "The Committee",
+            style = MaterialTheme.typography.headlineMedium,
+            fontFamily = FontFamily(Font(R.font.sf_pro_display_regular)),
+            fontWeight = FontWeight.Bold,
+            modifier =
+                Modifier.padding(horizontal = 16.dp, vertical = 10.dp).clickable {
+                  navigationActions.navigateTo(
+                      Destinations.COMMITTEE_DETAILS.route +
+                          "/${assoId}" +
+                          "/${association.acronym}")
+                })
+        Spacer(modifier = Modifier.height(50.dp))
+      }
```

Comment:
Maybe this should be a button rather than a text so that it is clear to the user that he can click on it and see the committee list